### PR TITLE
Custom row height functionality

### PR DESCRIPTION
**Pull request by GitMadridista**
Added a custom row height function

### CODE REVIEW

File: fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java [hunk 1]
```diff
@@ -762,15 +786,24 @@ private void writeFreezePane(Writer w) throws IOException {
...
+            w.append(" hidden=\"true");
+        }
+        if(customHeight) {
+        	w.append(" ht=\"")
+        	 .append(rowHeight)
+        	 .append("\"")
+        	 .append("customHeight=\"1\"");
         }
-        w.append("\">");
+        w.append(">");
```
Comment (ochedru) [inline fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java:806]: This breaks if `isHidden` is `true` and no custom height is set: closing quotes are missing.

File: fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java [hunk 2]
```diff
@@ -762,15 +786,24 @@ private void writeFreezePane(Writer w) throws IOException {
...
      * @param isHidden Whether or not this row is hidden
+     * @param customHeight Whether custom row height is set
+     * @param rowHeight Row height value in points to be set if customHeight is true
      * @param row Cells in the row.
      * @throws IOException If an I/O error occurs.
      */
-    private static void writeRow(Writer w, int r, boolean isHidden, Cell... row) throws IOException {
-        w.append("<row r=\"").append(r + 1);
+    private static void writeRow(Writer w, int r, boolean isHidden,
+    		boolean customHeight, double rowHeight, Cell... row) throws IOException {
```
Comment (ochedru) [inline fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java:795]: You could pass a single parameter of type `Double` or `Optional<Double>` for `rowHeight`?

Comment (GitMadridista) [reply fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java]: done

File: fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java (hunk 1 above)
Comment (GitMadridista) [reply fastexcel-writer/src/main/java/org/dhatim/fastexcel/Worksheet.java:806]: fixed it.

**Comment by ochedru** [comment]
Thanks for contributing!

**Pull request merged**